indico/modules/events/contributions/templates/display/_user_contribution_list.html

```diff
@@ -6,27 +6,27 @@
                     <h3>{{ title }}</h3>
                 </div>
             </div>
-            {{ _render_contribution_list(contributions) }}
+            {{ render_contribution_list(contributions) }}
         </section>
     {%- endif %}
 {% endmacro %}
 
-{% macro _render_contribution_list(contributions) %}
+{% macro render_contribution_list(contributions, enable_actions=True) %}
     <div class="i-box no-padding">
         <div class="i-box-content">
-            <ul class="group-list no-content-before with-buttons">
+            <ul class="group-list no-content-before with-buttons" style="margin-top: -10px;">
                 {% for contrib in contributions | sort(attribute='title') %}
-                    {{ _render_contribution_row(contrib) }}
+                    {{ _render_contribution_row(contrib, enable_actions=enable_actions) }}
                 {%- endfor %}
             </ul>
         </div>
     </div>
 {% endmacro %}
 
-{% macro _render_contribution_row(contrib) %}
+{% macro _render_contribution_row(contrib, enable_actions=True) %}
```

Comment:
Should this be `True` or `true`?

Reply:
`true`; jinja recommends that but both work